src/localconf.js

```diff
@@ -157,6 +157,16 @@ function buildConfig(opts) {
     explorerConfig = config.explorerConfig;
   }
 
+  // Two tiered-access options: site-wide and index-scoped.
+  // Tiered access is index-scoped if all guppyConfigs in the portal config
+  // contain a tierAccessLevel.
+  let indexScopedTierAccessMode = true;
+  explorerConfig.forEach((item) => {
+    if (!item.guppyConfig || !item.guppyConfig.tierAccessLevel) {
```

Comment:
minor: probably should just freak out if there is no `guppyConfig`, that block is required for each explorer tab

Reply:
the localconf.js file does not throw any errors or intentionally abort builds. i'm gonna keep with that convention